isBigInt.js

```diff
@@ -0,0 +1,27 @@
+import getTag from './.internal/getTag.js'
+
+/**
+ * Checks if `value` is classified as a `BigInt` primitive.
+ *
+ * @since 4.0.0
+ * @category Lang
+ * @param {*} value The value to check.
+ * @returns {boolean} Returns `true` if `value` is a BigInt, else `false`.
+ * @example
+ *
+ * isBigInt(1n)
+ * // => true
+ *
+ * isBigInt(BigInt('1'))
+ * // => true
+ *
+ * isBigInt(123)
+ * // => false
+ */
+
+function isBigInt(value) {
+  const tag = getTag(value)
```

Comment:
Not sure if this variable is needed.
Rest LGTM ✅

Reply:
Maybe, you are right!
I just want to reuse the `tag` for the future extension.

Reply:
Thanks for your code review.